cpp/tests/io/parquet_chunked_reader_test.cu

```diff
@@ -254,70 +254,70 @@ TEST_F(ParquetChunkedReaderTest, TestChunkedReadBoundaryCases)
   // Test with zero limit: everything will be read in one chunk
   {
     auto const [result, num_chunks] = chunked_read(filepath, 0);
-    EXPECT_EQ(num_chunks, 1);
+    // EXPECT_EQ(num_chunks, 1);
```

Comment:
This check can stay since we are reading with zero limits so everything should be read in one chunk